test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.Protocol.cs

```diff
@@ -565,6 +566,66 @@ public class Protocol
                     await connection.DisposeAsync().OrTimeout();
                 }
             }
+
+            [Fact]
+            public async Task ClientPingsMultipleTimes()
+            {
+                var connection = new TestConnection();
+                var hubConnection = CreateHubConnection(connection);
+
+                hubConnection.TickRate = TimeSpan.FromMilliseconds(30);
+                hubConnection.PingInterval = TimeSpan.FromMilliseconds(80);
+
+                try
+                {
+                    await hubConnection.StartAsync().OrTimeout();
+
+                    var firstPing = await connection.ReadSentTextMessageAsync().OrTimeout(TimeSpan.FromMilliseconds(200));
+                    Assert.Equal("{\"type\":6}", firstPing);
+
+                    var secondPing = await connection.ReadSentTextMessageAsync().OrTimeout(TimeSpan.FromMilliseconds(200));
+                    Assert.Equal("{\"type\":6}", secondPing);
+                }
+                finally
+                {
+                    await hubConnection.DisposeAsync().OrTimeout();
+                    await connection.DisposeAsync().OrTimeout();
+                }
+            }
+
+            [Fact]
+            public async Task ClientPingsStopIfSendingOtherMessages()
```

Comment:
As we discussed in person, let's just remove this test. It's too flaky and it doesn't test a high-value scenario (if we end up pinging despite other messages being sent, it's no big deal)